src/app/util/attribute-storage.cpp

```diff
@@ -82,6 +82,10 @@ GENERATED_FUNCTION_ARRAYS
 constexpr const chip::CommandId generatedCommands[] = GENERATED_COMMANDS;
 #endif // GENERATED_COMMANDS
 
+#if (defined(GENERATED_EVENTS) && (GENERATED_EVENT_COUNT > 0))
```

Comment:
When would this count ever be 0?  The Basic Information cluster is required and its StartUp event is required....

Reply:
Currently it is set to zero in:
`zzz_generated/darwin/controller-clusters/zap-generated/endpoint_config.h`

Only `client` version of the Basic Information cluster is enabled, which doesn't support events from `client --> server`.

Reply:
Ah, for a setup with no server clusters at all.  OK, that makes sense